src/sinopiaServer.js

```diff
@@ -95,3 +111,31 @@ export const getResourceTemplate = (templateId, group) => {
 
   return getResourceTemplateFromServer(templateId, group)
 }
+
+export const getGroups = () => {
+  if (Config.spoofSinopiaServer)
+    return new Promise(resolve => {
+      resolve({
+        response: {
+          body: {
+            contains: 'http://spoof.trellis.io/ld4p'
+          }
+        }
+      })
+    })
+
+  return new Promise(resolve => {
+    resolve(instance.getBaseWithHttpInfo())
+  })
+}
+
+export const listReourcesInGroupContainer = (group) => {
```

Comment:
Misspelling of function name, should this be `listResourcesInGroupContainer` instead?

Reply:
Ha! At least it was internally consistent! Fixed it.